### DISCUSSION TITLE
Polymorphic relationship (user has different profiles)

### DISCUSSION THREAD

**Discussion by theprobugmaker** [prisma/prisma#9448] [Q&A]
As Prisma doesn't have a polymorphic relationship support, what would be the best way to accomplish that an user can have one profile, s/he can either be a teacher or a student, each profile has different information.

Thank you.

**Comment by theprobugmaker** [comment]
Does anyone knows this?

**Comment by theprobugmaker** [comment]
It's been a while and no answers :(

**Reply by ryands17** [reply]
I don't think this is an easy approach as of now with Prisma as it doesn't support this like Rails does and you would need raw queries to perform joins. If the associations are not in a large quantity, I would suggest having separate tables for easier querying with Prisma.

**Reply by ryands17** [reply]
This is something that would interest you so a 👍 on this request would be great so that we know the priority for this: https://github.com/prisma/prisma/issues/2505

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
